apps/accounts/templates/accounts/change_secret_report.html

```diff
@@ -159,4 +156,181 @@ <h2>
+        width: 100%;
+    }
+
+    .header-container h2 span {
+        display: inline-flex;
+        margin-left: 8px;
+        font-weight: normal;
+    }
+
+    .triangle-right-icon {
+        display: inline-block;
+        width: 0;
+        height: 0;
+        border-top: 5px solid transparent;
+        border-bottom: 5px solid transparent;
+        border-left: 8px solid #1f2329;
+        margin-right: 10px;
+        cursor: pointer;
+    }
+
+    .triangle-down-icon {
+        display: inline-block;
+        width: 0;
+        height: 0;
+        border-left: 5px solid transparent;
+        border-right: 5px solid transparent;
+        border-top: 8px solid #1f2329;
+        margin-right: 10px;
+        cursor: pointer;
+    }
+
+    .container-section {
+        display: flex;
+        flex-wrap: wrap;
+        width: 100%;
+        height: 100%;
+        row-gap: 16px;
+        column-gap: 10px;
+    }
+
+    .collapsible-content {
+        overflow: hidden;
+        max-height: 1000px;
+        opacity: 1;
+        transition: max-height 0.3s ease-in-out, opacity 0.3s ease-in-out;
+    }
+
+    .collapsible-content.collapsed {
+        max-height: 0;
+        opacity: 0;
+        margin: 0;
+        padding: 0;
+    }
+
+    .container-section .item {
+        display: flex;
+        flex-basis: calc(33.33% - 10px);
+        flex-direction: column;
+        justify-content: flex-start;
+        align-items: flex-start;
+        gap: 4px;
+        padding: 4px 0;
+    }
+
+    .container-section .item .item-label {
+        color: #646a73;
+        font: 400 14px/22px 'PingFang SC';
+    }
+
+    .container-section .item .item-value {
+        color: #1f2329;
+        font: 500 14px/22px 'PingFang SC';
+    }
+
+
+    .custom-table {
+        margin-bottom: 0;
+        width: 100%;
+        border-collapse: collapse;
+        overflow: hidden;
+    }
+
+    .custom-table th {
+        background-color: #f5f7fa;
+        color: #646A73;
+        font: 500 14px/22px 'PingFang SC';
+        text-align: left;
+        padding: 12px 8px;
+        border-bottom: 1px solid #ebeef5;
+    }
+
+    .custom-table td {
+        color: #1F2329;
+        font: 400 14px/22px 'PingFang SC';
+        padding: 12px 8px;
+        border-bottom: 1px solid #ebeef5;
+    }
+
+    .custom-table tr:nth-child(even) {
+        background-color: #f6f6f6;
+    }
+
+    .custom-table tr:last-child td {
+        border-bottom: none;
+    }
+
+    .no-data {
+        color: #909399;
+        font: 400 14px/22px 'PingFang SC';
+        text-align: center;
+        padding: 20px;
+        font-style: italic;
+        background-color: #f9f9f9;
+        border-radius: 4px;
+        margin: 10px 0;
+        width: 100%;
+    }
+
+    .new-accounts .section-header span {
+        background: #e8f5e9;
+        color: #2e7d32;
+    }
+
+    .lost-accounts .section-header span {
+        background: #fbe9e7;
+        color: #d84315;
+    }
+    </style>
```

Comment:
There is no need to mention "current date" since you've mentioned the current year (2025), which indicates there won't be frequent updates from now onwards, so there shouldn't be much change in design or functionality over the years.

In summary:
1. Layout and structure remain largely unchanged.
2. The stylesheet seems fine without changes needed.
3. Javascript files should still be optimized for performance if required modifications occur in future versions of the software.